cherryontech/unmatched
# Feature/um 13 tag filter

## Description



## Related Tickets & Documents


## Acceptance Criteria
[x]Filter options are displayed below the search bar.

[x]When I select one or more filters, the displayed profiles are narrowed down to match the selected criteria.

[x]Clearing the filters or selecting a new filter updates the search results accordingly.

[x]The number of results displayed adjusts based on the applied filters.



## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|  x   | :sparkles: New feature     |
|     | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|  x   | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings
<img width="1149" alt="Screenshot 2023-11-03 at 11 46 34 AM" src="https://github.com/cherryontech/unmatched/assets/110904882/64d89cca-0c38-4f1f-85cc-2f1fa74f4cd7">



## What I learned


## Testing steps
Switch to this branch with the following command: git switch feature/UM-13-tag-filter
Install dependencies with the following command: npm i
Run the following scripts: npm run dev to start Vite's dev build & npm run tailwind to start the Tailwind CSS build
View the app in browser (the npm run dev script will return the url you can see the app at)
Navigate to the the Resources page and click on a tag to retrieve cards associated with it
Verify that the results displayed matches the tag clicked
Click on another tag to see results update accordingly 


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|     | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
